### PR TITLE
Slime Crossbreeding 2!

### PR DESCRIPTION
**Pull request by fludd12**
The [Insert Branch Name Here].

:cl:Fel
add: The other 12 slime crossbreeds are finally available! See the wiki for the new effects!
tweak: Love potions (and valentines day) give a floaty heart effect around your loved one! Cute!
tweak: Gold stabilized extracts now always produce the same pet, unless you squeeze them and change it. You can also now sentience potion them, and their ghost will inhabit any subsequent mobs spawned by it.
fix: Self-sustaining extracts are now available after an oversight prevented their use.
fix: Telekinesis can now be used to interact with machines from a distance as (apparently?) originally intended.
/:cl:

Slimes don't like the cold! Now that the freeze is over, this is coming up.

This miiiight take a while to code. Will provide updates both here and in the comments below.

Things that were modified that are seemingly unrelated but I had good reason to:
- There was a (completely unreported) issue in human canUseTopic which used AND instead of OR for a check. Now that it's fixed, telekinesis will probably have a lot less limitations. Whoops. Fixed to allow interaction with machines that do not require people to be close to them (i.e internal autolathe).
- I gave the love potion's heart effect to valentines antagonists. Mostly because that's really just kinda cute.
- There are now support for alternate appearances that only one person should be able to see, without that person having to be part of a predefined group.

### CODE REVIEW

**Comment by fludd12** [comment]
Shoot, bash didn't tell me about these conflicts. Will fix soon.

**Comment by fludd12** [comment]
Desperately hoping what I did fixes the conflict and doesn't make my diff a billion pages long.

**Comment by fludd12** [comment]
Well, shit.

File: tgstation.dme [hunk 1]
```diff
@@ -13,7 +13,7 @@
 // END_PREFERENCES
 
 // BEGIN_INCLUDE
-#include "_maps\_basemap.dm"
+#include "_maps\runtimestation.dm"
```
[the remark this answers is not in the document]
Comment (fludd12) [reply tgstation.dme]: Yepyep, had to do a quick commit so that progress wasn't lost when fixing things.

[the remark this answers is not in the document]
Comment (fludd12) [reply tgstation.dme]: Fixed.

**Comment by adamkad1** [comment]
bag reproductive feeding is broken

**Comment by fludd12** [comment]
Thank you for reminding me, will try to fix. It's probably because of Kevinz' storage refactor.

**Comment by fludd12** [comment]
Travis, why?

File: tgstation.dme (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jalleo) [reply tgstation.dme]: But this is still set here.

[the remark this answers is not in the document]
Comment (fludd12) [reply tgstation.dme]: `i do not know why`

**Comment by fludd12** [comment]
God fucking dammit, Travis. Everything compiles. Everything runs. What more do you want from me?

**Comment by kevinz000** [comment]
Besure to update (and give me lol) the doc :p

**Comment by arsserpentarium** [comment]
Yes, can you describe new changes more preciesly?

**Comment by TheDreamweaver** [comment]
Everything has been continuously documented here: https://docs.google.com/spreadsheets/d/1FhIHP_wEb1h1ir5if-k26ct9mnmcNgSaTSYy6nknoP4/edit#gid=0

**Comment by newfren** [comment]
What ones are being added in this PR?

**Comment by TheDreamweaver** [comment]
@newfren According to Fel, they plan on completing them all before this is merged.

**Comment by TheDreamweaver** [comment]
The greyed out ones are the ones already coded.

**Comment by XDTM** [comment]
I don't mind this but you're adding more content to the job that already has a huge amount of content compared to other jobs

**Comment by Shadowflame909** [comment]
When is more (optional) things to do a bad thing? Infact, I'm glad they want all this optional content. It's like a dlc for xenobio! I wish medical had this much (OPTIONAL!) content to do. Would make it a less boring job.

**Comment by TheDreamweaver** [comment]
Shadow beat me to it, but ya, we shouldn't be complaining about more options. If anything, it means that some of the many metas that have built up over time in the game will finnally be broken. And if your discontent with the amount of content in other departments in comparison, maybe use that as fuel to finnally give them more content? (*cough cough* medical and service departments)

**Comment by Shadowflame909** [comment]
What dreamweaver says, I would love more optional content anywhere. But if its forced content onto jobs, forcing me to do more. I feel like it's more of an unfun timegate.

**Comment by Shadowflame909** [comment]
Are reproductive extracts self sustaining? Because I think reproductive extracts are br0ke

**Comment by XDTM** [comment]
Doctor crossbreeding?

File: code/modules/research/xenobiology/crossbreeding/_status_effects.dm [hunk 2]
```diff
@@ -57,6 +57,145 @@
...
+	owner.cut_overlay(bluespace)
+	if(interrupted || !ismob(target))
+		to_chat(owner, "<span class='warning'>The bluespace tug fades away, and you feel that the force has passed you by.</span>")
+		return
+	owner.visible_message("<span class='warning'>[owner] disappears in a flurry of sparks!</span>",
+		"<span class='warning'>The unknown force snatches briefly you from reality, and deposits you next to [target]!</span>")
+	do_sparks(3, TRUE, owner)
+	owner.forceMove(target.loc)
+
+/obj/screen/alert/status_effect/freon/stasis
```
[the remark this answers is not in the document]
Comment (TheDreamweaver) [reply code/modules/research/xenobiology/crossbreeding/_status_effects.dm]: Oh mei-god

[the remark this answers is not in the document]
Comment (fludd12) [reply code/modules/research/xenobiology/crossbreeding/_status_effects.dm]: Dòng zhù! Bùxǔ zǒu!

[the remark this answers is not in the document]
Comment (adamkad1) [reply code/modules/research/xenobiology/crossbreeding/_status_effects.dm]: stop invoking satan

[the remark this answers is not in the document]
Comment (Shadowflame909) [reply code/modules/research/xenobiology/crossbreeding/_status_effects.dm]: Should have a 1 percent chance to blood rage the person being summoned. Good way to backfire on the summoner.

[the remark this answers is not in the document]
Comment (fludd12) [reply code/modules/research/xenobiology/crossbreeding/_status_effects.dm]: If you mean for slime recall, it's more intended to be a way to organize people - chilling is the defensive extract type. Link your friends, then recall them when they're in danger or when your nerdfort is getting cracked down on by sec.

**Comment by fludd12** [comment]
Still working on symbiont. Who'd've thought that adding 22 unique organs with an average of 1-2 abilities each might take a while?

Notable symbiont organs:
- The Industrial Heart gives you an internal autolathe that can't be hacked or upgraded.
- The Jolt Foot lets you blink forwards 3 spaces.
- The Chronal Foot lets you teleport to where you were 5 seconds ago, or kick someone else there!
And, my personal favorite
- The Warping Palm lets you pick up any item you can see, with a fancy beam effect - not only that, but throwing with that hand also places the item wherever you click!

**Comment by vuonojenmustaturska** [comment]
conflicts

**Comment by fludd12** [comment]
`shite`

**Comment by optimumtact** [comment]
hey

**Comment by fludd12** [comment]
Sorry for the intense delays on any updates, finals are coming and they are hitting me hard. I'll reopen this when I've made more progress.

**Comment by TimeIsEndless** [comment]
Is there any progress being made on this?

**Pull request closed without merging**